src/showcase/app/business/business-examples/status-examples/status-custom-icon-example/status-custom-icon-example.component.html

```diff
@@ -0,0 +1,3 @@
+<sbb-status class="cloud" aria-label="It is cloudy today">
```

Comment:
General question: what is the difference between sbb-status custom icon and just use a custom icon with a background color?